week3/src/components/CommonPage.jsx

```diff
@@ -0,0 +1,105 @@
+import styled from "styled-components";
+import { useEffect, useState, useMemo } from "react";
+import SingleCard from "./SingleCard";
+import Header from "./Header";
+import Modal from "./Modal";
+import ModalPortal from "./ModalPortal";
+
+const CommonPage = (cardList) => {
+  const [choiceOne, setChoiceOne] = useState(null);
+  const [choiceTwo, setChoiceTwo] = useState(null);
```

Comment:
각각 state로 만들지 않고 요런식으로 배열로 관리하는건 오때?!
```suggestion
  const [choices, setChoices] = useState([]);
```

Reply:
옹 참고해보께!